WORKSPACE

```diff
@@ -6,6 +6,17 @@ workspace(name = "carbon")
 
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
+skylib_version = "1.3.0"
+
+http_archive(
+    name = "bazel_skylib",
+    urls = [
+        "https://mirror.bazel.build/github.com/bazelbuild/bazel-skylib/releases/download/{version}/bazel-skylib-{version}.tar.gz".format(version=skylib_version),
```

Comment:
I've been trying to keep the formatting in bazel files consistent, and currently using %s below. Would it make sense to switch? (if you want to switch all the others, I'll accept that, I just want it to be consistent)

Reply:
I have no strong opinion really -- mostly the examples used this form and it needs the parameter to be expanded *twice*, not once, and I don't know enough python to know how to do that cleanly with `%s`.

Let me know what you prefer, I'm very happy w/ any approach here.

Reply:
I'll note, %s formatting would actually be shorter overall: while you're right that `skylib_version` would be repeated, the current approach is more verbose due to repetition of `version`. Also, repeating would be consistent with https://github.com/carbon-language/carbon-lang/blob/trunk/WORKSPACE#L206

But, I've sent you #2433 if you're okay with that approach, I'm wondering if you maybe didn't realize the naming was optional.

Reply:
I've merged #2433 and incorporated that change into here, so should be GTG here?